macros/json_value.sql

```diff
@@ -13,5 +13,9 @@
 {%- endmacro -%}
 
 {%- macro snowflake__json_value(column, path, array, dtype) -%}
-  GET_PATH(to_variant({{column}}), '{{ modules.re.sub('\$\.', '', path) }}')
```

Comment:
Was this not working? We shouldn't have to parse_json in snowflake since it should already be in a VARIANT column?

Reply:
That's what the `_shim.sql` file was for in the integration models. Doesn't feel like we should  be writing our common macros based on our integration test formulations.

Reply:
@camphillips22 all fixed now.

Reply:
Nice! Is there a reason to avoid using GET_PATH? The change is fine, just curious.

Reply:
Not really a reason to avoid, but I felt like the colon syntax was a bit easier to read in the outputted query (eventually on the docs site). `GET_PATH` is typically reserved for when you have a dynamic input or are using a variable as the second argument in order to specify the field name. Since we know everything we're looking for, this seemed like a cleaner alternative.